File: 序号/344.反转字符串.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size(),m=n/2;
        for(int i = 0;i<m;i++){
            s[i]=s[i]+s[n-1-i];
            s[n-1-i]=s[i]-s[n-1-i];
            s[i]=s[i]-s[n-1-i];
        }
    }
};

File: 序号/344. 反转字符串.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        l,r=0,len(s)-1
        while l<r:
            ll=s[l]
            s[l]=s[r]
            s[r]=ll
            l,r=l+1,r-1
        return s
